litex/boards/platforms/fomu_hacker.py

```diff
@@ -30,10 +41,14 @@
         Subsignal("clk", Pins("D1"), IOStandard("LVCMOS33")),
         Subsignal("mosi", Pins("F1"), IOStandard("LVCMOS33")),
         Subsignal("miso", Pins("E1"), IOStandard("LVCMOS33")),
+        Subsignal("wp",   Pins("A1"), IOStandard("LVCMOS33")),
+        Subsignal("hold", Pins("B1"), IOStandard("LVCMOS33")),
```

Comment:
I didn't think the `wp` / `hold` signals were actually connected on the hacker board?